node/src/chain_spec/darwinia.rs

```diff
@@ -136,7 +136,33 @@ pub fn local_config() -> ChainSpec {
 }
 
 pub fn genesis_config() -> ChainSpec {
-	// TODO: update this before the final release
+	let collators = [
+		(
+			"0x196f03b77a1acd0db080006b04d2f3a991ebbe68",
+			"0xa05255010ee986b9684a444d10a74aa0ecbe781f5002e871665add894752cc7e",
+		),
+		(
+			"0x4D3D5958B948e8d749FaB0236b179fCC55d9aAc0",
```

Comment:
This is my collator address and session key. I don't have enough machine to run another parachain node. Please delete it.